### PR TITLE
metal sheet exploit fix

### PR DESCRIPTION
**Pull request by kiVts**
# About the pull request

funny thing I noticed while surfing the code
# Explain why it's good for the game
Im pretty sure thats not how world works
# Testing Photographs and Procedure
<details>
<summary>Screenshots & Videos</summary>

Put screenshots and videos here with an empty line between the screenshots and the `<details>` tags.

</details>

# Changelog
:cl: kivts
fix: Infinite metal oversight fix.
/:cl:

### CODE REVIEW

**Comment by kiVts** [comment]
print table parts in autolathe > click with wood, this gives one metal sheet for "replacing metal parts with wood" > feed the wood parts into autolathe = +1 net positive on metal.

**Comment by Diegoflores31** [comment]
> print table parts in autolathe > click with wood, this gives one metal sheet for "replacing metal parts with wood" > feed the wood parts into autolathe = +1 net positive on metal.

not exactly "infinite metal" rather an oversight of inheritance

**Review by Drulikar — COMMENTED** [review]
Why not
`matter = list("wood" = 2000)`
Or whatever amount it ought to be?

File: code/modules/projectiles/gun_attachables.dm [hunk 1]
```diff
@@ -1761,7 +1761,7 @@ Defined in conflicts.dm of the #defines folder.
 	//but at the same time you are slow when 2 handed
 	aim_speed_mod = CONFIG_GET(number/slowdown_med)
 
-	matter = list("wood" = 2000)
+	matter = null
```
Comment (Drulikar) [inline code/modules/projectiles/gun_attachables.dm:1765]:
```suggestion
```

**Review by Drulikar — APPROVED** [review]

**Pull request merged**